ambry-protocol/src/main/java/com/github/ambry/protocol/ReplicateBlobRequest.java

```diff
@@ -0,0 +1,171 @@
+/**
+ * Copyright 2022 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.protocol;
+
+import com.github.ambry.clustermap.ClusterMap;
+import com.github.ambry.commons.BlobId;
+import com.github.ambry.utils.Utils;
+import java.io.DataInputStream;
+import java.io.IOException;
+
+
+/**
+ * ReplicateBlob request to replicate one particular blob
+ */
+public class ReplicateBlobRequest extends RequestOrResponse {
+  public final static short ReplicateBlob_REQUEST_VERSION_1 = 1;
```

Comment:
I would suggest to keep it consistent as other request's naming pattern and use capital letters. REPLICATE_BLOB_REQUEST_VERSION_1.

Reply:
Updated.

Reply:
Oh, just saw it. Sophie, you are right. Other classes are using the long name. But it also makes sense to make it shorter since the variable's scope is inside the ReplicateBlobRequest.